src/ray/gcs/tables.h

```diff
@@ -142,6 +141,71 @@ class Table {
   int64_t subscribe_callback_index_;
 };
 
+/// \class Table
+///
+/// A GCS table where every entry is a single data item.
+template <typename ID, typename Data>
+class Table : private Log<ID, Data> {
```

Comment:
"All inheritance should be public. If you want to do private inheritance, you should be including an instance of the base class as a member instead."

http://google.github.io/styleguide/cppguide.html#Implementation_Comments

Reply:
In this case, I think it actually makes sense to use private inheritance because the `Table` implementation is very close to that of the `Log`, except that it should expose an `Add` method instead of an `Append.